Python/Python.sublime-syntax

```diff
@@ -519,14 +516,48 @@ contexts:
       pop: true
 
   decorators:
-    - match: '^\s*(?=@)'
+    - match: ^\s*(?=@)
       push:
-        - meta_content_scope: meta.statement.decorator.python
+        # Due to line continuations, we don't know whether this is a "function call" yet
+        - meta_content_scope: meta.annotation.python
         - match: '@'
-          scope: keyword.other.decorator.python
+          scope: punctuation.definition.annotation.python
+        - match: $
+          pop: true
         - include: line-continuation-or-pop
-        - include: function-calls
-        - include: dotted-name
+        - match: (?=\.?\s*{{path}}\s*\() # now we do
+          set: [decorator-function-call-wrapper, qualified-name-until-leaf]
+        - match: (?=\.?\s*{{path}})
+          push: [decorator-wrapper, qualified-name-until-leaf]
```

Comment:
So this (and two lines above) is where you are pushing two contexts, the later of which does not consume anything?